deploy/wip-catalog/README.md

```diff
@@ -0,0 +1,75 @@
+# Service Catalog WIP Helm Chart
+
+The Helm Chart deploys Service Catalog into an existing Kubernetes cluster.
+
+## Prerequisites
+
+  - [Helm](https://github.com/kubernetes/helm) must be installed in the cluster.
+  - Images must be built from source and pushed to a registry accessible to
+    the cluster if deploying to a non-local cluster. Example (run from this
+    directory):
+
+    ```
+    $ export REGISTRY=hub.docker.com/<username>
+    $ make -C ../.. push
+    ```
+
+    __Note:__ If deploying locally (e.g. in Minikube), it can be more efficient
+    to use a _local_ Docker registry.
+
+## Usage
+
+Supported template parameters (values):
+
+### Deployment Knobs
+
+- `registry`: Container registry with Service Catalog images; optional; defaults
+  to unset
+- `version`: Version of Service Catalog (container images) to deploy; optional;
+  defaults to unset
+- `namespace`: A Kubernetes namespace to use for deployment of this
+  chart into k8s; optional; defaults to `default`
+
+### Catalog knobs
+
+- `verbosity`: The verbosity of logs; optional; defaults to `10`
+- `debug`: Whether to create a load balancer for the apiserver and
+  controller-manager services; optional; defaults to `false`
+- `insecure`: Whether the API server should serve insecurely; optional; defaults
+  to `false`
+- `imagePullPolicy`: The image pull policy to use for all pods in this chart;
+  optional; defaults to `Always`
+
+#### API Server knobs
+
+- `insecure`: Whether the API server should serve insecurely; optional; defaults
+  to `false`
+- `insecurePort`: When `insecure` is true, the API server pod serves insecurely
+  on this container port; optional; defaults to `8081`
+- `insecureServicePort`: When `insecure` is true, the API server is fronted by a
+  k8s service serving on this node port; optional; defaults to `30001`
+- `storageType`: The type of storage for the API server to use; optional;
+  defaults to `etcd`
+- `globalNamespec`: The namespace to store global resources in when the API
+  server is backed by k8s Third Party Resources.
```

Comment:
also, add a `controllerManagerVersion` -- maybe that should go into a new "Controller Manager Knobs" section?